docker-compose.yml

```diff
@@ -183,7 +183,9 @@ services:
       - SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus
   rails:
     env_file: .env
-    build: .
+    build:
+      context: .
+      dockerfile: devel.Dockerfile
```

Comment:
Why not with the suffix, `Dockerfile.devel`, which is more common?

Reply:
I was about to say it's because of the filetype detection, but popular Vim plugins for Dockerfile syntax seem to support any version as long as they have "dockerfile" anywhere in the name

https://github.com/ekalinin/Dockerfile.vim/blob/master/ftdetect/Dockerfile.vim#L4-L8

personally I don't have strong opinions on any format :shrug:

Reply:
There is no convention, and it seems to be by design: https://github.com/moby/moby/issues/15549

However, I've actually seen more of this format than what you suggest, Viliam, mostly due to file naming conventions where the extension indicates the file type (and editors/IDEs recognize extensions).

I also do not feel strongly about this, and I hope we can remove this specific devel Dockerfile in the near future.